src/modules/article.js

```diff
@@ -0,0 +1,114 @@
+import axios from "axios";
+
```

Comment:
there seems t be a lot of methods in this module that are out of scope?

Reply:
Yes, I seem to accidentally have done two of the features in one.